src/Network/Network/help/New-AzNetworkManagerRoutingRule.md

```diff
@@ -0,0 +1,308 @@
+---
+external help file: Microsoft.Azure.PowerShell.Cmdlets.Network.dll-Help.xml
+Module Name: Az.Network
+online version: https://learn.microsoft.com/powershell/module/az.network/new-aznetworkmanagerroutingrule
+schema: 2.0.0
+---
+
+# New-AzNetworkManagerRoutingRule
+
+## SYNOPSIS
+Creates a routing rule.
+
+## SYNTAX
+
+### Custom (Default)
+```
+New-AzNetworkManagerRoutingRule -Name <String> -RuleCollectionName <String>
+ -RoutingConfigurationName <String> -NetworkManagerName <String> -ResourceGroupName <String>
+ [-Description <String>]
+ [-Destination <PSNetworkManagerRoutingRuleDestination>]
```

Comment:
are destination and nextHop not required?

Reply:
Fixed this in the recent commit.